.pre-commit-config.yaml

```diff
@@ -1,26 +1,43 @@
 exclude: ^web/static/vendored/.+$
 repos:
-    - repo: https://github.com/pre-commit/pre-commit-hooks
-      rev: v4.0.1
-      hooks:
-          - id: check-merge-conflict
-          - id: end-of-file-fixer
-          - id: mixed-line-ending
-            args: [--fix=no]
-          - id: trailing-whitespace
-    - repo: https://github.com/PyCQA/isort
-      rev: 5.9.3
-      hooks:
-          - id: isort
-    - repo: https://github.com/psf/black
-      rev: 21.9b0
-      hooks:
-          - id: black
-    - repo: https://github.com/pycqa/flake8
-      rev: 3.9.2
-      hooks:
-          - id: flake8
-            additional_dependencies:
-                - flake8-bugbear~=21.4
-                - flake8-pytest-style~=1.5
-                - pep8-naming~=0.11
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+        args: [--fix=no]
+      - id: trailing-whitespace
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v8.2.0
+    hooks:
+      - id: eslint
+        additional_dependencies:
+          - eslint@8.2.0
+          - eslint-config-standard@16.0.3
+          - eslint-config-prettier@8.3.0
+          - eslint-plugin-import@2.25.3
+          - eslint-plugin-node@11.1.0
+          - eslint-plugin-promise@5.1.1
```

Comment:
It doesn't seem like the eslintrc is configured to use these plugins. Or does it use them implicitly. What's the reason for adding these?

Reply:
it does use explicitly with eslint-config-standard so I had to add them